### PR TITLE
Allows deconstruction of wide airlocks and cleans up the airlock code a bit

### LINKED ISSUE #4638: You can see through non-glass wide doors.
### Testmerges

#4575: 'Adds discretionary arrests as capital crime' by nauticall at commit 3c52b35869
[#4437](https://github.com/cmss13-devs/cmss13/pull/4437): 'Project ARES: Datacore' by realforest2001 at commit 64e77569e7

### Description of the bug

When the wide doors are opened, you can see through one of the tiles even if the doors aren't solid.

### What's the difference with what should have happened?

You should not be able to see through solid airlocks.

### How do we reproduce this bug?

1. Find solid, non-glass wide doors
2. Double check that they are in fact solid.
3. Open the doors.
4. Close the doors.
5. Realise you can still see through them.

### Issue Bingo

- [X] Issue could be reproduced at least once
- [X] Issue happened in a recent (less than 7 days ago) round
- [X] Couldn't find an existing issue about this (https://github.com/cmss13-devs/cmss13/issues)

### PR DESCRIPTION
**Pull request by Das15**
# About the pull request

Allows disassembly of wide airlocks and fixes the bugs which... caused it to be disabled in the first place. Since moving multi-tile objects is a bit... inconsistent at best, unwrenching of wide airlock assemblies is disabled. 

Also renamed a lot of single-letter variables and added sprites to wide airlock assembly and generic solid wide airlock since for some reason I couldn't find it.

And fixes #4638.
# Explain why it's good for the game

Inconsistent behavior bad. Unable to disassemble wide airlocks bad. Able to disassemble all airlocks good.

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

https://media.discordapp.net/attachments/703724778825187332/1163117987943944242/wide_airlock.mp4

https://cdn.discordapp.com/attachments/703724778825187332/1163151828251324536/wide_airlock_solid_v2.mp4

</details>

# Changelog
:cl: MrDas
fix: You can now disassemble wide airlocks. (You can't unwrench them though)
spellcheck: Fixed a few typos in airlock assembly.
imageadd: Added wide airlock assembly and generic solid wide airlock sprites.
/:cl:

### CODE REVIEW

**Comment by LC4492** [comment]
You see, while at that, you could take a check on a bug that occurs with those wide airlock: you are able to see through the ones without windows after they are used once in the round. I believe it have something to do with only one of the tiles that the airlock take do block the vision, with the other being just blank.

**Comment by Das15** [comment]
> You see, while at that, you could take a check on a bug that occurs with those wide airlocks: you are able to see through the ones without windows after they are used once in the round. I believe it have something to do with only one of the tiles that the airlock take block the vision, with the other being just blank.

Yup, I've accidently found out what caused that bug... It was the lack of . = ..() in multi_airlock/almayer/handle_multidoor override. I've already patched it out.

File: code/game/objects/structures/airlock_assembly.dm [hunk 1]
```diff
@@ -108,10 +112,14 @@
 
 	switch(state)
 		if(STATE_STANDARD)
-			if(HAS_TRAIT(W, TRAIT_TOOL_WRENCH))
+			if(HAS_TRAIT(attacking_item, TRAIT_TOOL_WRENCH))
+				//Moving wide doors is wonky and doesn't work properly, if it's fixed we could make it unwrenchable again
+				if(width > 1 && anchored)
+					to_chat(user, SPAN_WARNING("\The [src] cannot be unwrenched."))
```
Comment (harryob) [inline code/game/objects/structures/airlock_assembly.dm:118]:
```suggestion
					to_chat(user, SPAN_WARNING("[src] cannot be unwrenched."))
```

File: code/game/objects/structures/airlock_assembly.dm [hunk 2]
```diff
@@ -238,107 +242,107 @@
 	else
 		icon_state = "door_as_[base_icon_state][state]"
 
+/obj/structure/airlock_assembly/proc/get_airlock_path()
+	//For some reason multi_tile doors have different paths... can't say it isn't annoying
+	if (width > 1)
+		return "/obj/structure/machinery/door/airlock/multi_tile/almayer/[airlock_type][glass ? "" : "/solid"]"
+	return "/obj/structure/machinery/door/airlock/almayer/[airlock_type][glass ? "/glass" : ""]"
+
+//Used for overloading proc in multi_tile
```
Comment (harryob) [inline code/game/objects/structures/airlock_assembly.dm:251]:
```suggestion
/// Used for overloading proc in multi_tile
```
this turns the comment into a dm doc

File: code/game/objects/structures/airlock_assembly.dm [hunk 3]
```diff
@@ -22,9 +22,10 @@
 	var/airlock_type = "generic" //the type path of the airlock once completed
 	var/glass = AIRLOCK_NOGLASS // see defines
 	var/created_name = null
+	var/width = 1 //Used for multitile assemblies
```
Comment (harryob) [inline code/game/objects/structures/airlock_assembly.dm:25]:
```suggestion
	/// Used for multitile assemblies
	var/width = 1 
```
i know this isn't the same as above, but this is the correct way of annotating variables (it shows up correctly in vscode this way)

**Review by harryob — APPROVED** [review]

**Pull request merged**